viper.py

```diff
@@ -670,6 +693,9 @@ def fit_chunk(o, chunk, obsname, targ=None, tpltarg=None):
     for mol in range(0, len(molec),1):
         w_atm[mol], f_atm[mol] = Tell(molec[mol])
 
+# collect all file for createtpl function
+tpl_all = defaultdict(dict)
```

Comment:
I think, `sp_all` is a more precise name than `tpl_all`.

Reply:
Ok, I will change.